openapi-cli/src/main/java/io/ballerina/openapi/generators/service/ServiceDiagnosticMessages.java

```diff
@@ -31,10 +31,20 @@
             "supported in Ballerina.", DiagnosticSeverity.ERROR),
     OAS_SERVICE_101("OAS_SERVICE_101",
             "Query parameters with no array item type can not be mapped to Ballerina resource" +
-                    " query parameters.",
-            DiagnosticSeverity.ERROR),
+                    " query parameters.", DiagnosticSeverity.ERROR),
     OAS_SERVICE_102("OAS_SERVICE_102",
             "Query parameters with type '%s' can not be mapped to the Ballerina query parameters.",
+            DiagnosticSeverity.ERROR),
+    OAS_SERVICE_103("OAS_SERVICE_103", "Header '%s' with '%s' array item type are not " +
```

Comment:
Shouldnt we use `''%s''` syntax for the template args?

Reply:
Yeah we need to use `''%s''` but this error message append to OpenAPIExeption message due to that it didn't ignore first `'` 
here. I will change this into `''%s''` once we will use diagnostic error handling. 🙂